beams/beams/custom_scripts/job_applicant/job_applicant.py

```diff
@@ -99,18 +99,16 @@ def send_magic_link(applicant_id):
     """
     doc = frappe.get_doc('Job Applicant', applicant_id)
     link = generate_magic_link(doc.name)
-
-    subject = f"{doc.applicant_name}, Complete Your Application"
+    template = frappe.get_doc("Email Template", "Job Applicant Follow Up")
```

Comment:
Always use frappe.db.exists() before frappe.db.get_doc(), In this case make sure to give a warning that the template does not exist